V0.12.0

jcapriot opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

jcapriot committed:
rerender with latest conda-smithy

jcapriot committed:
update versions

jcapriot committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.08.21.44.29

jcapriot committed:
add threaded migrations

jcapriot committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.08.21.44.29

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

jcapriot committed:
bump minimum scipy

jcapriot committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.08.21.44.29

Other tools:
- conda-build 24.9.0
- rattler-build 0.27.0
- rattler-build-conda-compat 1.4.9

jcapriot committed:
remove 3.14t migrator for now

jcapriot committed:
touch is_freethreading inside meta.yml

jcapriot committed:
add migrator back

jcapriot committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.08.21.44.29  Other tools: - conda-build 24.9.0 - rattler-build 0.27.0 - rattler-build-conda-compat 1.4.9

jcapriot committed:
fix recipe

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found some lint.

Here's what I've got...


For **recipe/meta.yaml**:

 * ❌ Selectors are suggested to take a ``<two spaces>#<one space>[<expression>]`` form. See lines [15]


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/18388832738](https://github.com/conda-forge/conda-forge-webservices/actions/runs/18388832738). Examine the logs at this URL for more detail.</sub>

jcapriot committed:
adjust ordering?

jcapriot committed:
switch to rattler-build

jcapriot committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.08.21.44.29

Other tools:
- conda-build 25.9.0
- rattler-build 0.48.1
- rattler-build-conda-compat 1.4.9

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/recipe.yaml**:

 * ℹ️ Found `bld.bat` in recipe directory, but this is a recipe v1 (rattler-build recipe). rattler-build uses `build.bat` instead of `bld.bat` for Windows builds. Consider renaming `bld.bat` to `build.bat`.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/18389551113](https://github.com/conda-forge/conda-forge-webservices/actions/runs/18389551113). Examine the logs at this URL for more detail.</sub>

jcapriot committed:
rename bld.bat

jcapriot committed:
add pixi file

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

jcapriot committed:
exclude 3.14 threaded on aarch64 and ppc64le
